### ISSUE TITLE
Latest or-tools source code build fails for dotnet in stable and master branch

### ISSUE THREAD

**Issue by gurubhg** [google/or-tools#2733]
**What version of OR-Tools and what language are you using?**
Latest or-tools source code build fails for dotnet in stable and master branch
Version: master/v9.0
Language: C#

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
GLOP

**What operating system (Linux, Windows, ...) and version?**
Windows

**What did you do?**
or-tools build fails for dotnet in stable branch:
tools\make dotnet
.\ortools\graph\max_flow.cc(927): error C2374: 'kMaxFlowQuantity': redefinition; multiple initialization
.\ortools\graph\max_flow.cc(927): note: see declaration of 'kMaxFlowQuantity'
.\ortools\graph\max_flow.cc(928): note: while compiling class template static data member 'const operations_research::FlowQuantity operations_research::GenericMaxFlow<operations_research::SimpleMaxFlow::Graph>::kMaxFlowQuantity'
.\ortools\graph\max_flow.cc(78): note: see reference to class template instantiation 'operations_research::GenericMaxFlow<operations_research::SimpleMaxFlow::Graph>' being compiled
make: *** [makefiles/Makefile.gen.mk:1156: objs/graph/max_flow.obj] Error 2

or-tools build fails for dotnet in master branch:
tools\make test_dotnet
Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:01.33
"C:\Program Files\dotnet\dotnet.EXE" run --no-build  --project ortools\\sat\\samples\\ChannelingSampleSat.csproj
Check failed: !parameters.enumerate_all_solutions() Enumerating all solutions in parallel is not supported.
*** Check failure stack trace: ***
    @   00007FFF8C9CAD05  (unknown)
    @   00007FFF8C9C9309  (unknown)
    @   00007FFF8CA61A8E  (unknown)
    @   00007FFF8CA5F7D5  (unknown)
    @   00007FFF8C9322D2  (unknown)
    @   00007FFF35A100B5  (unknown)
make[1]: *** [makefiles/Makefile.dotnet.mk:513: run] Error -1073740791
make[1]: Leaving directory 'C:/Users/guhiriya/Downloads/or-tools-master'
make: *** [makefiles/Makefile.dotnet.mk:635: test_dotnet_sat_samples] Error 2

**What did you expect to see**
Success build

**What did you see instead?**
Build failure

Make sure you include information that can help us debug (full error message, model Proto).

**Anything else we should know about your project / environment**
N/A

**Comment by Mizux** [comment]
This is fixed on master, we should create a new release end of this month or beginning of September 2021

**Comment by Mizux** [comment]
duplicate of #2557
